src/test/controllers/users.controller.test.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
```

Comment:
ES6 doesn't require this: http://stackoverflow.com/questions/31685262/not-recommended-to-write-out-use-strict-with-es6